platforms/hyperledger-indy/configuration/roles/clean/vault/tasks/main.yaml

```diff
@@ -0,0 +1,136 @@
+        exit 1
+      fi
+    }
+    response_status=$(curl -o /dev/null -s -w "%{http_code}\n" --header "X-Vault-Token: ${vault_token}" --request DELETE {{ vault.url }}/v1/sys/policy/{{ serviceItem.name }}-ro)
+    validateVaultResponse ${response_status}
+  loop: "{{ services.trustees }}"
+  loop_control:
+    loop_var: serviceItem
+  when: services.trustees is defined
+
+# Remove Policies of stewards
+- name: Remove Policies of stewards
+  environment:
+    vault_token: "{{ vault.root_token }}"
+  shell: |
+    validateVaultResponse () {
+      if [ ${1} != 204 ]; then
+        echo "ERROR: Unable to retrieve. Http status: ${1}"
+        exit 1
+      fi
+    }
+    response_status=$(curl -o /dev/null -s -w "%{http_code}\n" --header "X-Vault-Token: ${vault_token}" --request DELETE {{ vault.url }}/v1/sys/policy/{{ serviceItem.name }}-ro)
+    validateVaultResponse ${response_status}
+  loop: "{{ services.stewards }}"
+  loop_control:
+    loop_var: serviceItem
+  when: services.stewards is defined
+
+# Remove Policies of endorsers
+- name: Remove Policies of endorsers
+  environment:
+    vault_token: "{{ vault.root_token }}"
+  shell: |
+    validateVaultResponse () {
+      if [ ${1} != 204 ]; then
+        echo "ERROR: Unable to retrieve. Http status: ${1}"
+        exit 1
+      fi
+    }
+    response_status=$(curl -o /dev/null -s -w "%{http_code}\n" --header "X-Vault-Token: ${vault_token}" --request DELETE {{ vault.url }}/v1/sys/policy/{{ serviceItem.name }}-ro)
+    validateVaultResponse ${response_status}
+  loop: "{{ services.endorsers }}"
+  loop_control:
+    loop_var: serviceItem
+  when: services.endorsers is defined
+
+# Remove Policies of organization
+- name: Remove Policies of {{ organization }}
+  environment:
+    vault_token: "{{ vault.root_token }}"
+  shell: |
+    validateVaultResponse () {
+      if [ ${1} != 204 ]; then
+        echo "ERROR: Unable to retrieve. Http status: ${1}"
+        exit 1
+      fi
+    }
+
+    response_status=$(curl -o /dev/null -s -w "%{http_code}\n" --header "X-Vault-Token: ${vault_token}" --request DELETE {{ vault.url }}/v1/sys/policy/{{ organization }}-ac-ro)
+    validateVaultResponse ${response_status}
+    response_status=$(curl -o /dev/null -s -w "%{http_code}\n" --header "X-Vault-Token: ${vault_token}" --request DELETE {{ vault.url }}/v1/sys/policy/{{ organization }}-admin-rw)
+    validateVaultResponse ${response_status}
+
+# Remove Kubernetes Authentication Methods
+- name: Remove Kubernetes Authentication Methods of {{ organization }}
+  vars:
+    auth_path: "kubernetes-{{ organization }}"
+  environment:
+    vault_token: "{{ vault.root_token }}"
+  shell: |
+    validateVaultResponse () {
+      if [ ${1} != 204 ]; then
+        echo "ERROR: Unable to retrieve. Http status: ${1}"
+        exit 1
+      fi
+    }
+    response_status=$(curl -o /dev/null -s -w "%{http_code}\n" --header "X-Vault-Token: ${vault_token}" --request DELETE {{ vault.url }}/v1/sys/auth/{{ auth_path }})
```

Comment:
For every org there are separate auth methods for every identity + the admin-auth + the baf-ac-auth, please make sure all of them are deleted.